bauble/src/value/asset_context.rs

```diff
@@ -0,0 +1,234 @@
+#[derive(Debug, Clone)]
+pub enum DataFields {
```

Comment:
I think this would be easier to understand if it was called `FieldsKind` or something like that, since it's called `kind` in most of the places it's used.